app/models/card_grant.rb

```diff
@@ -139,6 +139,28 @@ def topup!(amount_cents:, topped_up_by: User.find(sent_by_id))
     end
   end
 
+  def withdrawl!(amount_cents:, withdrew_from_by: User.find(sent_by_id))
```

Comment:
```suggestion
  def withdraw!(amount_cents:, withdrew_from_by: User.find(sent_by_id))
```
Sorry - this is pedantic - let's use the verb `withdraw` for the method etc.

Reply:
I knew I was gonna be tripping over myself with that lol